fourth-blog-post.html

```diff
@@ -0,0 +1,150 @@
+      <h1>Kyoto VR MQP Team Blog</h1>
+
+      <h2>Week 5 & 6 (8/19 - 8/30): Oh Yeah, It's All Coming Together</h2>
+
+      <time class='blogdate' datetime="2019-08-30">August 30, 2019</time>
+
+      <br>
+
+      <span class="author">Will Campbell</span>
+
+      <p>
+      <img class="blogimage" src="images/kronk.jpg"
+      alt="When your project starts to look and work like an actual app">
+      </p>
+      <p>
+        Our project is really starting to come together! It's been
+        all thanks to the huge amount of progress we've made over the past
+        couple weeks. With Week 5, we knew it was time to really start pushing
+        towards a presentable alpha product. In our initial timeline, the end of
+        Week 6 is about when we stated we would have our alpha complete. At the
+        start of Week 5, we knew we had a lot of work to do to take all of the
+        cool and important features we had been making and combine them into
+        something that looked like a real application. So on the application
+        side of things, we got started working on what would be the User
+        Interface for our alpha application, and potentially the final product.
+      </p>
+
+      <p>
+        The first step in this process was simply learning how Unity's UI worked,
+        and what features it offered for UI design. After spending some time
+        researching and testing different things, we began development of the
+        alpha UI. We knew that the app would be used on smartphones, so we
+        needed to put some thought into layouts that would be comfortable to
+        interact with on a standard phone screen. We also needed to bring
+        together all of the features and UI elements that we had been creating
+        separately over the past few weeks. In some cases this was as easy as
+        dragging and dropping the Unity assets from one scene into the next,
+        like with our Audio Slider and Play/Pause buttons for audio tour files.
+        In other cases, this meant creating an actual UI element for a feature
+        that had previously been implemented without one, like our buttons for
+        swapping which AR content was on display. By the end of the week, we
+        had a UI that we were really proud of.
+      </p>
+
+      <img class="blogimage" src="images/ui.jpg"
+      alt="Our UI that we've been working on for our alpha">
+
+      <p>
+        Lastly, coming out of week 5, we also implemented a new and important
+        feature for AR content. Atticus explained that as AR content goes, he
+        would like to primarily show off images to the user, like of traditional
+        art and the like. So to allow for this and integrate with the Editour
+        web app we made for creating tours, we created a feature that takes the
+        images you add to a tour in the web app, and then creates AR content of
+        them by creating a Plane Primitive in Unity, assigning the image as a
```

Comment:
I would get rid of the capitalization. We know these are classes but they are not really proper nouns.